src/components/views/rooms/RoomSublist.tsx

```diff
@@ -60,6 +60,11 @@ export const HEADER_HEIGHT = 32; // As defined by CSS
 
 const MAX_PADDING_HEIGHT = SHOW_N_BUTTON_HEIGHT + RESIZE_HANDLE_HEIGHT;
 
+const ALWAYS_VISIBLE_TAGS: TagID[] = [
+    DefaultTagID.DM,
+    DefaultTagID.Untagged,
+];
+
```

Comment:
this feels like the sort of thing that should still be in the RoomList component (or appropriate model once we get there) - we can pass a `forceVisible` prop though, or if we can word it without using the word "force" then even better.

Reply:
> I'm not sure how to interpret the screenshot: it looks like we're spending _more_ time in code?

I've read my description again, and it is utterly confusing, the left figure is AFTER my improvements. and the right figure is BEFORE 👀 

Jonathan (jboi) also took a recording and there, the left figure is BEFORE and the right figure is AFTER (sorry this is all very confusing)
![Screen Shot 2021-04-15 at 19 49 20](https://user-images.githubusercontent.com/769871/114988416-382dc380-9e8e-11eb-8203-5529766a0af9.png)

> Overall this looks sensible, but I am curious how it interacts with the tab indexing/accessibility.

Generally speaking screen readers will ignore everything that is `display: none;`

> It'd also be good to get a comment somewhere in the code to say we deliberately keep the sublists rendered so we don't accidentally optimize it back out 😅

Will do 👍